elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -160,14 +160,11 @@ class PfeNavigation extends PFElement {
     if (state) {
       // Add the overlay to the page
       this._overlay.removeAttribute("hidden");
-      // This prevents background scroll while nav is open
-      document.body.style.overflow = "hidden";
```

Comment:
This doesn't look quite right to me - is this the latest?  Without the overflow hidden on the body, doesn't the scroll move behind the overlay?